jenkins/e2e-image/e2e-runner.sh

```diff
@@ -243,8 +243,10 @@ function setup_gci_vars() {
 
 ### Pre Set Up ###
 if running_in_docker; then
```

Comment:
Rather than making it even harder to understand the different set of things that happen depending on whether or not we call this from inside a container I would prefer we remove this check entirely.

Like, why aren't awscli and gcloud already on our e2e image? Why do we need to spend the time, bandwidth, stability and comprehensibility installing these two utilities thousands of times per hour?